### DISCUSSION TITLE
Silvercrest Door- and Windowsensor - Cannot rejoin to network

### DISCUSSION THREAD

**Discussion by sparky-Ol** [Koenkk/zigbee2mqtt#14402] [Devices]
Hi,
i have installed 3 [HG06336](https://www.zigbee2mqtt.io/devices/HG06336.html) in my network. First pairing and installation went fine but after a while two sensors lost connection (no more mqtt-messages). I wanted to dis- and reconnect the devices but that worked only at the device. I zigbee2mqtt the disconnection failed:
`Oct 08 11:45:31 raspberrypi npm[2320]: Zigbee2MQTT:error 2022-10-08 11:45:31: Failed                                                                                                                                 to remove Schlafzimmerfenster (Error: AREQ - ZDO - mgmtLeaveRsp after 10000ms)`
Now i cannot rejoin the network. When i put the devices in connection mode nothing happens at zigbee2mqtt. Even removing the batteries didn't fix it.
How can i remove a device by cli?
Does anybody has a different idea?

Sascha

**Comment by McMornan** [comment]
Hi, I suggest to remove those sensors using the gui (force remove). Otherwise you can use the mqtt commands as per the documentation. I have the same issues with these sensors. They are disconnected every time I need to change batteries. Deleting and adding them again seems to work ...
